### ISSUE TITLE
Outdated Packages(?)

### ISSUE THREAD

**Issue by ft-scobra** [Beannsss/PlexAniSync#19]
Can't install embypython since it doesn't exist (anymore?). 
![image](https://github.com/user-attachments/assets/b2e4281a-4447-4829-81e9-c63357717c36)

Tried replacing embypython w/ embyapi at every instance it was used but I ran into issues. 
![image](https://github.com/user-attachments/assets/a1cde4b4-c07b-4578-a766-a07a4be54e97)

**Comment by ft-scobra** [comment]
NVM, I had downloaded from releases.

**Comment by bpwhelan** [comment]
Yeah, I'm not a fan of how I have emby's client library just in the repo as source code, I'd much rather have it installed with pip, but i ran into many issues during development.

Keeping this open to maybe take another look at redoing this.

**Comment by bpwhelan** [comment]
https://github.com/bpwhelan/EmbyAniSync/issues/23
